expression/builtin_info_vec_test.go

```diff
@@ -33,10 +33,12 @@ var vecBuiltinInfoCases = map[string][]vecExprBenchCase{
 	ast.Database: {
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{}},
 	},
-	ast.User:          {},
-	ast.TiDBDecodeKey: {},
-	ast.RowCount:      {},
-	ast.CurrentRole:   {},
+	ast.User: {},
+	ast.TiDBDecodeKey: {
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString}},
```

Comment:
Could you add a new test and implement a new `gener` reflecting the specific input string of `builtinTiDBDecodeKeySig`